build-step-npm/action.yml

```diff
@@ -57,7 +58,7 @@ runs:
       shell: bash
       run: |
         cd ${{ inputs.module_path }}
-        yarn pack
+        yarn jahia-pack
```

Comment:
can't we make it both compatible ? check if jahia-pack available then use pack as default? (or the way around)

Reply:
That's already sort of what jahia-pack does. I don't understand the problem of requiring jahia-pack in projects. Also how do you suggest we test this?